apps/backoffice-v2/src/domains/workflows/hooks/queries/useWorkflowDefinitionQuery/useWorkflowDefinitionQuery.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from '@tanstack/react-query';
+import { workflowsQueryKeys } from '../../../query-keys';
+
+export const useWorkflowDefinitionQuery = ({
+  workflowDefinitionId,
+}: {
+  workflowDefinitionId: string;
+}) => {
+  return useQuery({
+    ...workflowsQueryKeys.workflowDefinition({ workflowDefinitionId }),
```

Comment:
Ideally there'll be `workflowDefinitionQueryKeys` so the structure can remain the same of `list` and `byId` and open the possibility of creating and updating definitions without mixing into runtime.